Modules/Core/Common/include/itkDerivativeOperator.h

```diff
@@ -102,9 +99,11 @@ class ITK_TEMPLATE_EXPORT DerivativeOperator : public NeighborhoodOperator<TPixe
   }
 
 protected:
-  /** Typedef support for coefficient vector type.  Necessary to
-   * work around compiler bug on VC++. */
-  using CoefficientVector = typename Superclass::CoefficientVector;
+  /** Type alias support for pixel real type.*/
+  using typename Superclass::PixelRealType;
```

Comment:
If PixelRealType is public in superclass, and we make it protected here, will it cause problems in possible descendants?

Reply:
Although no class inherits from `itk::DerivativeOperator`, I put it back to the public section.